app/views/favorite/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @heading %></h1>
+<%= link_to "Remove All", "/favorites", method: :delete %>
+<% @pets.each do |pet| %>
+<article id="pet-<%= pet.id %>-details">
+  <img><%= image_tag(pet.image, width: "200") %></img>
+  <h2>Name: <%= link_to "#{pet.name}", "/pets/#{pet.id}"%></h2>
+  <ul>
+    <li><%= link_to 'Remove', "/favorites/#{pet.id}", method: :delete %>
+  </ul>
+  <br>
+</article>
+<% end %>
```

Comment:
The use of id's here is nice. We should add the same functionality in the shelters and pets index.